### DISCUSSION TITLE
How did you update this to work with the newer unity versions?

### DISCUSSION THREAD

**Discussion by Kavukamari** [aelurum/AssetStudioMod#42] [Q&A]
I noticed that this version is more up to date than the master, I was wonder if the same process of updating this could be done with Asset Bundle Extractor to be able to pack and unpack to the same version this one does? do you know how I might be able to write a patch or a Type Database for this version of unity?

to be more specific with what exactly I need: Do you have a list of all of the Resource types along with their Type IDs? I figured out how to make my own type database, I just need the IDs

**Comment by aelurum** [comment]
- There is an updated UABE, it is called UABEA (https://github.com/nesrak1/UABEA).
- AssetStudio doesn't use pre-generated type tree dumps (*.tpk) for asset parsing, compared to most other tools, including UABE. All code for parsing is written manually but kinda relies on type tree dumps since recently.
- https://github.com/AssetRipper/TypeTreeDumps, https://github.com/AssetRipper/Tpk
